examples/fluids/smartsim_regression_framework.py

```diff
@@ -13,7 +13,6 @@
 import os
 import shutil
 import logging
-import socket
```

Comment:
Looks like most of the changes in this file are from a failed merge. They should be removed.

Reply:
To be clear, `import socket`  should be added back in.

Reply:
This should be resolved in latest commits